src/vma/sock/sockinfo.cpp

```diff
@@ -1172,17 +1165,15 @@ void sockinfo::rx_del_ring_cb(flow_tuple_with_local_if &flow_key, ring* p_ring,
 			delete p_ring_info;
 
 			if (m_p_rx_ring == base_ring) {
+				/* Remove event from rx ring if it is active
+				 * or just reinitialize
+				 * ring should not have events related closed socket
+				 * in wait list
+				 */
+				m_p_rx_ring->del_ec(&m_socketxtreme.ec);
```

Comment:
Do we want to check if socketxtreme is enabled ?

Reply:
It is redundant as far as nothing is done in case VMA is in legacy mode.